charts/uptime-kuma/templates/configmap.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.extraCertificates.enabled }}
+{{- if and .Values.extraCertificates.enabled (not .Values.extraCertificates.existingObject) }}
```

```suggestion
{{- if and .Values.extraCertificates.enabled (not .Values.extraCertificates.existingConfigMap) }}
```

Unless I am missing something, but I think this is a typo

---

Yep, had a different idea before (allow secrets, configmaps, ...) but that made the change way more complicated. Switched to CM-only and forgot this place. Will update the PR!